### PR TITLE
Update whats-new.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the steps for import and export settings catalog policies

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4358

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please review and sign off this PR. Thanks

**Comment by Brenduns** [comment]
#sign-off

**Review by Brenduns — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: memdocs/intune/fundamentals/whats-new.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Applies to:
 - Android device administrator (DA)
 
 #### Import and export settings catalog policies<!-- 3470151  -->  
-The Intune [settings catalog](../configuration/settings-catalog.md) lists all the settings you can configure, and all in one place (**Devices** > **Configuration profiles** > **Create profile** > Select your **platform** > For **Profile**, select **Settings catalog**).
+The Intune [settings catalog](../configuration/settings-catalog.md) lists all the settings you can configure, and all in one place (**Devices** > **Configuration profiles** > **Create** > **New Policy** >Select your **platform** > For **Profile**, select **Settings catalog**).
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/whats-new.md:179]:
```suggestion
The Intune [settings catalog](../configuration/settings-catalog.md) lists all the settings you can configure, and all in one place (**Devices** > **Configuration profiles** > **Create** > **New Policy** > Select your **platform** > For **Profile**, select **Settings catalog**).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Pull request merged**